backend/archiver/config/variables.py

```
@@ -102,6 +103,10 @@ def MINIO_REGION(self) -> str:
     def MINIO_ENDPOINT(self) -> str:
         return self.__get("minio_endpoint")
 
+    @property
+    def MINIO_URL_EXPIRATION_DAYS(self) -> int:
+        return int(self.__get("minio_url_expiration_days"))
```

If I understood it correctly, the `__get` method returns an empty string if the variable has not been defined. We probably should throw an error instead (follow-up issue).

---

The tests agree
```
utils/log.py:27: in wrapper_decorator
    value = func(*args, **kwargs)
utils/s3_storage_interface.py:71: in get_presigned_url
    ExpiresIn=Variables().MINIO_URL_EXPIRATION_DAYS*days_to_seconds,  # URL expiration time in seconds
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <archiver.config.variables.Variables object at 0x7f62d0c32[51](https://github.com/SwissOpenEM/ScopeMArchiver/actions/runs/14904703268/job/41864303133?pr=191#step:6:52)0>

    @property
    def MINIO_URL_EXPIRATION_DAYS(self) -> int:
>       return int(self.__get("minio_url_expiration_days"))
E       ValueError: invalid literal for int() with base 10: ''
```